src/autofill-utils.js

```diff
@@ -69,45 +70,68 @@ const setValueForInput = (el, val) => {
     originalSet.call(el, val)
     events.forEach((ev) => el.dispatchEvent(ev))
     el.blur()
+
+    return true
```

Comment:
As mentioned above, I started to return booleans to avoid highlighting fields we couldn't autofill. Useful for `select` elements where we couldn't find a suitable match.